charmhelpers/contrib/network/files/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2014-2015 Canonical Limited.
```

Comment:
Please either update this to 2014-2022, or remove the copyright entirely (it's an empty file)